README.md

```diff
@@ -65,6 +65,13 @@ from a given starting state.
 StateMachine.verify(Awake).isRight() shouldBe true
 ```
 
+#### Document your state machine
+
+The utility `StateMachine.mermaid` will generate a mermaid diagram of your state machine. This can be rendered in markdown.
+The diagram of HamsterStates above was created using this utility.
+
+```kotlin
```

Comment:
Do you need closing tags here?

Reply:
It looks a little bit broken. 
![Screenshot 2024-01-25 at 3 34 45 pm](https://github.com/cashapp/kfsm/assets/16330962/f9411fa4-3ed5-4e12-ba7f-6d4c911c1ee2)

Reply:
TY